packages/account/src/Sections/Security/ConnectedApps/data-table-template.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { toMoment } from '@deriv/shared';
+import { Button } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+const getConnectedAppsColumnsTemplate = (handleToggleModal) => [
+    { title: localize('Name'), col_index: 'name' },
+    {
+        title: localize('Permission'),
+        col_index: 'scopes',
+        renderCellContent: ({ cell_value }) => {
+            return prepareConnectedAppsScopes(cell_value);
+        },
+    },
+    {
+        title: localize('Last login'),
+        col_index: 'last_used',
+        renderCellContent: ({ cell_value }) => prepareConnectedAppsLastLogin(cell_value),
+    },
+    {
+        title: localize('Action'),
+        col_index: 'app_id',
+        renderCellContent: ({ cell_value }) => prepareConnectedAppsAction(cell_value, handleToggleModal),
+    },
+];
+
+const prepareConnectedAppsAction = (app_id, handleToggleModal) => {
```

Comment:
Can we rename all the names of components in this file to be in pascal case to follow React's component naming standards?

Reply:
@arash-fs Please also change the file extension to `jsx`.